project/src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+public final class User {
+    private final long id;
+    private final String userEmail;
+    private final String userName; 
+    private final String userPassword;
+  
+    //public User(long id, String userName, String userEmail, String userPassword)
+     public User(long id, String userName, String userEmail, String userPassword) {
```

Comment:
We can use this ID to link the user to predefined workout routines?

Reply:
Yes, this unique ID will be used for that purpose.